packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2485,8 +2485,14 @@
   "Livechat_offline": "Omnichannel offline",
   "Livechat_offline_message_sent": "Livechat offline message sent",
   "Livechat_OfflineMessageToChannel_enabled": "Send Livechat offline messages to a channel",
+  "livechat_on_hold_chat_resumed": "On Hold Chat Resumed: __comment__",
```

Comment:
All translation keys here should start with `Omnichannel_` rather than `Livechat_`.

Reply:
Done